langchain4j-core/src/main/java/dev/langchain4j/data/pdf/PdfFile.java

```diff
@@ -0,0 +1,143 @@
+package dev.langchain4j.data.pdf;
+
+import dev.langchain4j.Experimental;
+
+import java.net.URI;
+import java.util.Objects;
+
+import static dev.langchain4j.internal.Utils.quoted;
+
+@Experimental
+public class PdfFile {
+    private final URI url;
+    private final String base64Data;
+    private final String mimeType;
```

Comment:
```suggestion
```
No need in mimeType as it is always `application/pdf`, right?

Please remove it from all builders/ctors here and in `PdfFileContent`